apps/ledger-live-desktop/src/renderer/components/Web3AppWebview/helpers.ts

```diff
@@ -166,18 +170,49 @@ export function useWebviewState(
 
   const handleDomReady = useCallback(() => {
     const webview = webviewRef.current;
-
     if (!webview) {
       return;
     }
 
-    setState({
+    setState(oldState => ({
       url: webview.getURL(),
       canGoBack: webview.canGoBack(),
       canGoForward: webview.canGoForward(),
       title: webview.getTitle(),
       loading: webview.isLoading(),
-    });
+      isAppUnavailable: oldState.isAppUnavailable,
+    }));
+  }, [webviewRef]);
+
+  const handleFailLoad = (useCallback(
+    (errorCode: number, errorDescription: string, validatedURL: string, isMainFrame: boolean) => {
+      const errorInfo = { errorCode, errorDescription, validatedURL, isMainFrame };
+      console.error("Web3AppView handleFailLoad", errorInfo);
+
+      track("useWebviewState", errorInfo);
```

Comment:
Avoid sending too much "raw" data to tracker, for privacy concerns.

errorCode = no pb
errorDescription = we don't know what websites will put there, should be removed
validatedURL = we know there are wallet addresses for instance, please retain only the domain & subdomain (drop path, query params...)

Reply:
I've addressed this issue in new commit